sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -986,10 +986,13 @@ private async Task<AmqpSession> CreateAndOpenSessionAsync(AmqpConnection connect
             {
                 var openObjectCompletionSource = new TaskCompletionSource<object>(TaskCreationOptions.RunContinuationsAsynchronously);
 
-                using var registration = cancellationToken.Register(static state =>
+                using var registration = cancellationToken.Register(state =>
```

Comment:
You can pass a ValueTuple as state which then will be boxed unfortunately or use a regular Tuple